.github/actions/build/action.yml

```diff
@@ -27,6 +27,10 @@ inputs:
     type: string
     required: false
     default: linux/amd64,linux/arm64
+  SUB_DIRECTORY:
```

Comment:
This is more just for my info, but why did we need to add the subdirectory to prevent building on `ARM64`? I would have just assumed that we needed to change Platforms setting down below >.>

Reply:
Good question! The SUB_DIRECTORY input was mainly added to help point to specific test Dockerfiles in the testing workflow (for example [here](https://github.com/Dwolla/jenkins-agents-workflow/blob/JDK8/.github/workflows/test.yml#L12)).
Like you said, the PLATFORMS input was used for "major" change in this PR - `linux/amd64` is now the only platform specified for JDK8 specific steps, e.g. [here](https://github.com/Dwolla/jenkins-agents-workflow/blob/JDK8/.github/workflows/build-docker-image.yml#L139).

Reply:
Ok that makes sense! Thank you for answering my q :^)